yarn.lock

```diff
@@ -15544,18 +15544,18 @@ react-async-hook@^4.0.0:
   resolved "https://registry.yarnpkg.com/react-async-hook/-/react-async-hook-4.0.0.tgz#1f0467586654e1f33b7433bd98c300a0c5f9b3d0"
   integrity sha512-97lgjFkOcHCTYSrsKBpsXg3iVWM0LnzedB749iP76sb3/8Ouu4nHIkCLEOrQWHVYqrYxjF05NN6GHoXWFkB3Kw==
 
-react-devtools-core@4.26.0:
```

Comment:
This looks like it downgraded. This seems suspect with the [yarn lock check fail.](https://github.com/valora-inc/wallet/actions/runs/4766429475/jobs/8473430942?pr=3667)

Reply:
Otherwise this is working for me locally on iOS and Android.

Reply:
it didn't downgrade, if you scroll down you can see that it upgraded from 4.13 to 4.19 (just the order was switched so the diff looks a bit weird)

Reply:
the yarn.lock job uses `actions/cache@v3` but i don't think this is working as expected. i tried without this step and the job succeeded :/